subsys/settings/Kconfig

```diff
@@ -26,6 +26,14 @@ config SETTINGS_DYNAMIC_HANDLERS
 	help
 	  Enables the use of dynamic settings handlers
 
+config SETTINGS_COMMIT_PRIO_MINIMUM
+	int "minimum commit priority level"
+	default 0
+	help
+	  Minimum commit priority level. All h_commit from
```

Comment:
It is not clear which priority is higher, which lower. Could you elaborate description/

Reply:
I'm using the same convention as for thread priorities, higher number means lower priority.

Reply:
That's Ok. Could you add a phrase about this here?